### ISSUE TITLE
Local part of prefixed name may not start with hyphen-minus (-) or full stop (.)

### ISSUE THREAD

**Issue by sgoetz-brox** [atextor/owl-cli#4]
`test.ttl`:

```
@prefix ex: <https://example.org/> .

<https://example.org/-10> a <https://example.org/Test> .
```

`owl write test.ttl` generates:

```
@prefix ex: <https://example.org/> .

ex:-10 a ex:Test .
```

If I’m not mistaken, the generated Turtle is invalid, because the local part of a prefixed name must not start with `-`. 

(The same issue exists for `.`, and maybe also other characters.)

## Allowed first character in the local part of a prefixed name

If I read it correctly, the local part [may start with](https://www.w3.org/TR/2014/REC-turtle-20140225/#grammar-production-PN_LOCAL) `(PN_CHARS_U | ':' | [0-9] | PLX)`. Following the references, this seems to be the list of allowed start characters:

- `_`
- `:` 
- `%` (followed by _HEX_ _HEX_)
- `\` (followed by one of: `_`  `~` `.`  `-`  `!` `$` `&` `'` `(` `)` `*` `+` `,` `;` `=` `/` `?` `#` `@` `%`)
- [0-9]
- [A-Z]
- [a-z]
- [#x00C0-#x00D6]
- [#x00D8-#x00F6]
- [#x00F8-#x02FF]
- [#x0370-#x037D]
- [#x037F-#x1FFF]
- [#x200C-#x200D]
- [#x2070-#x218F]
- [#x2C00-#x2FEF]
- [#x3001-#xD7FF]
- [#xF900-#xFDCF]
- [#xFDF0-#xFFFD]
- [#x10000-#xEFFFF]

**Comment by atextor** [comment]
Hi @sgoetz-brox, thanks for reporting!
From preliminarily scanning the given sections I think you're right. (I might have to concentrate a bit more though, since I've been laying down with fever the past couple days and still do)

**Comment by atextor** [comment]
Fixed in [v1.2.3](https://github.com/atextor/owl-cli/releases/tag/v1.2.3).
